javascript-forms/form-events.js

```diff
@@ -0,0 +1,21 @@
+function handleFocus(event) {
+  console.log('focus fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('eventName:', event.target.name, 'eventValue:', event.target.value);
+}
+
+var focusEvent = document.querySelector('#user-name');
+var blurEvent = document.querySelector('#user-email');
+var inputEvent = document.querySelector('#user-message');
+
+focusEvent.addEventListener('focus', handleFocus);
+blurEvent.addEventListener('blur', handleBlur);
+inputEvent.addEventListener('input', handleInput);
```

Comment:
You need to apply all three event handlers to all three elements.
![image](https://user-images.githubusercontent.com/32880187/146466361-0918237b-0336-4ac1-9721-9d6643966e8d.png)